content/en/real_user_monitoring/guide/rum-for-product-analytics.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use RUM & Session Replay for Product Analytics
```

Comment:
can this also be the title that's shown in the nav? felt like the one in the preview was a bit too long